resources/splunk_app.rb

```diff
@@ -162,30 +163,40 @@ class PackagedApp < SplunkApp
 
   action :install do
     return unless !version || changed?(:version)
+    @converge_actions = nil # Clear resource updates by property change
```

Comment:
It's a little difficult... So with resources, they're given a desired property and I'm populating the existing property. In the case this is necessary, there is a 1.0.0-somesnapshotversion on disk, and the resource is given `version 1.0.0`. The app it downloads will have 1.0.0-somesnapshotversion which is the exact same version, and I do checks for this later in the resource to determine whether or not we need to upgrade the app (which would be a change in the resource, triggering notifications).

The problem is that Chef considers a difference in the given property and the existing property to be a "change", reports this in the output, and adds a "converge action" to this list. Since a difference in given properties isn't indicative of an actual change, I'm clearing the converge actions before doing anything else.